src/utils/expenses.js

```diff
@@ -0,0 +1,59 @@
+const { Op } = require('sequelize');
+
+const { getValidString } = require('./validation');
+const { getErrorWithStatus } = require('./getError');
+
+const isExpenseValid = (expense) => {
+  const { title, category, spentAt, note } = expense;
+
+  const textFields = {
+    title,
+    category,
+    spentAt,
+    note,
+  };
+
+  Object.entries(textFields).forEach(([key, value]) => {
+    const field = key.toString();
+
+    if (field === 'note' && !field) {
+      return;
+    }
+
+    getValidString(value, field);
+  });
+
+  if (typeof expense.amount !== 'number') {
+    throw getErrorWithStatus(400, `Type of amount must be string`);
```

Comment:
The error message for the `amount` field should state that the type must be a number, not a string. Consider changing it to `Type of amount must be number`.